src/ekorepp/anomalous_dimensions/unpolarized/space_like/as1.hpp

```diff
@@ -0,0 +1,21 @@
+#include "../../../constants.hpp"
+#include "../../../harmonics/w1.hpp"
+
+
+namespace ekorepp {
+namespace anomalous_dimensions {
+namespace unpolarized {
+namespace spacelike {
+namespace as1 {
```

Comment:
C++ looks terribly old...

Reply:
But definitely better than Numba!

Reply:
it's simply the fastest I can write after Python ...